src/Panel/Controller/Changes.php

```diff
@@ -84,9 +81,6 @@ public static function save(ModelWithContent $model, array $input): array
 			language: 'current'
 		);
 
-		// add the model to the user's list of unsaved changes
-		App::instance()->site()->changes()->track($model);
```

Comment:
`Question:` Where is the tracking happening now?

And is the untacking still needed in `Controller\Changes::publish()` as it would also be taken care by `Version::delete()` now?